tests/test_consumer.py

```diff
@@ -542,3 +540,20 @@ def test_consumer_group_without_subscription(self):
         yield from consumer.start()
         yield from asyncio.sleep(0.2, loop=self.loop)
         yield from consumer.stop()
+
+    @run_until_complete
+    def test_consumer_wait_topic(self):
+        topic = "some-test-topic-for-autocreate"
+        consumer = AIOKafkaConsumer(
+            topic, loop=self.loop, bootstrap_servers=self.hosts)
+        yield from consumer.start()
```

Comment:
I think this test should:
* Start consumer
* start a background `getone` task.
* wait ~0.5 seconds. Just to be sure `getone` does not fail
* produce 1 message
* check message